DOCS/compile-windows.md

```diff
@@ -212,15 +212,15 @@ For guidance on updating MSYS2, please refer to the official documentation:
 ### Installing mpv Dependencies
 
 ``` bash
-# Install pacboy
-pacman -S pactoys
+# Install pacboy and git
+pacman -S pactoys git
 
 # Install MSYS2 build dependencies and a MinGW-w64 compiler
-pacboy -S git {python,pkgconf,cc,meson}:p
+pacboy -S {python,pkgconf,cc,meson}:p
```

Comment:
Ok, so with newer pacboy we don't need to specify `:p`, so we could simplify it. to 
```suggestion
pacboy -S python pkgconf cc meson
```

Reply:
Did this and also simplified the second pacboy command too